cmake/Modules/FindSDL_mixer.cmake

```diff
@@ -104,21 +106,22 @@ find_package_handle_standard_args(SDL_mixer
                                   REQUIRED_VARS SDL_MIXER_LIBRARY SDL_MIXER_INCLUDE_DIR ${additionalVars}
                                   VERSION_VAR SDL_MIXER_VERSION)
 
-if(NOT TARGET SDL_mixer::SDL_mixer)
-  add_library(SDL_mixer-SDL_mixer INTERFACE)
-  add_library(SDL_mixer::SDL_mixer ALIAS SDL_mixer-SDL_mixer)
-  target_link_libraries(SDL_mixer-SDL_mixer INTERFACE ${SDL_MIXER_LIBRARY})
-  target_include_directories(SDL_mixer-SDL_mixer SYSTEM INTERFACE ${SDL_MIXER_INCLUDE_DIR})
+if(NOT TARGET SDL${SDL_MIXER_VERSION_MAJOR}_mixer::SDL_mixer)
+  add_library(SDL${SDL_MIXER_VERSION_MAJOR}_mixer-SDL_mixer INTERFACE)
```

Comment:
Use a variable for `SDL${SDL_MIXER_VERSION_MAJOR}_mixer-SDL_mixer` to avoid the repetition. It hard to read...